lib/pure/collections/sequtils.nim

```diff
@@ -103,31 +165,32 @@ proc cycle*[T](s: openArray[T], n: Natural): seq[T] =
 
 proc repeat*[T](x: T, n: Natural): seq[T] =
   ## Returns a new sequence with the item `x` repeated `n` times.
+  ## `n` must be a non-negative number (zero or more).
```

Comment:
Yes, and you will see more of these "not needed" comments. And the reason for it is because these modules are the ones that Nim beginners will probably first use, so I introduced some extra descriptions on purpose.

Reply:
Because why not. Could be the outcome of a more general computation.